PowerPointLabs/PowerPointLabs/SyncLab/SyncLabShapeStorage.cs

```diff
@@ -61,8 +62,18 @@ public string CopyShape(Shape shape, Format[] formats)
             {
                 try
                 {
-                    shape.Copy();
+                    // workabout for 2013's artistic effect, (2010 & 2016 do not require this)
+                    // copied shapes will have their artistic effect permenantly set to the shape
+                    // remove the artistic effect before copying, and restore it after pasting
+                    Shape duplicate = shape.Duplicate()[1];
+                    ArtisticEffectFormat.RemoveArtisticEffect(duplicate);
+                    
+                    duplicate.Copy();
```

Comment:
This will affect the clipboard right? Do you want to use the RestoreClipboardfAterAction function?